### DISCUSSION TITLE
建议 PaoPaoGateWay 增加支持运行容器的功能

### DISCUSSION THREAD

**Discussion by 9857389** [kkkgo/PaoPaoGateWay#148] [想法]
前阵子偶然在 GH 上看到大佬的项目，发现竟然是关注了很多年的大佬，兴冲冲的拿来使用，现在用 PaoPaoDNS+PaoPaoGateWay，效果堪称完美，那运行是相当丝滑。现在想到一种场景，建议大佬能否考虑一下，那就是 PaoPaoGateWay 能否支持运行容器，如果可以的话，就可以把 DNS 和 GateWay 整合在一个机器上从而实现 All In One 的效果了。这样子弄个小主机直接同时运行两个服务，感觉利用率会很高。

**Comment by kkkgo** [comment]
PaoPaoGateWay 设计就是无硬盘启动的，没有运行容器的环境。
你想all in one的话就装类似黑裙，unraid这种，有docker，也有虚拟机。

**Reply by 9857389** [reply]
看了一下openwrt-builder的代码，感觉确实是设计理念上确实是冲突的，感谢回复！
